src/test/java/systemtests/SystemTestSetupHelper.java

```diff
@@ -22,7 +22,7 @@
      */
     public TestApp setupApplication(Supplier<ReadOnlyAddressBook> addressBook, String saveFileLocation) {
         try {
-            FxToolkit.registerStage(Stage::new).setMaximized(true);
```

Comment:
Your code still has browserPanel so without this, you might fail the helpwindowtest

Reply:
there was no error when I checked with gradlew clean......
I edited this part for personal preference(prefer small screen haha). But I'll change this before merging PR for consistency.